editor/src/components/navigator/navigator-item/navigator-item.tsx

```diff
@@ -215,8 +116,8 @@ const collapseItem = (
   elementPath: ElementPath,
   e: React.MouseEvent<HTMLDivElement>,
 ) => {
-  dispatch([EditorActions.toggleCollapse(elementPath)], 'leftpane')
   e.stopPropagation()
+  dispatch([EditorActions.toggleCollapse(elementPath)], 'leftpane')
```

Comment:
Huh? What did this fix?

Reply:
oh – nothing really, my OCD while cruising through all the handlers desperately wanted to have that `e.stopPropagation()` at the beginning of the funciton like all others, since there should be no reason for it to follow the dispatch

Reply:
oh I thought there's one more browser API gotcha I need to keep in mind